rllib/agents/trainer.py

```diff
@@ -594,7 +596,6 @@ def get_scope():
                 extra_config.update({
                     "batch_mode": "complete_episodes",
                     "batch_steps": 1,
-                    "in_evaluation": True,
```

Comment:
Can we still force set this to avoid user error (or at least assert).

Reply:
Sure. Good idea.